packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -19,6 +19,7 @@ export class DdSdkReactNativeConfiguration {
      */
     public sampleRate?: number;
     public sessionSamplingRate: number = 100.0;
+    public tracingSamplingRate: number = 100.0;
```

Comment:
It seems I forgot that we also have `DdTrace` in the SDK (this module is rarely mentioned), when discussing property name. So basically we introduce this sampling only for RUM integration, so for the network requests.

We don't expect it to be applied to the spans created by `DdTrace` and we need to make it clear. With current name customer may take an assumption that `DdTrace` is also affected.

Out of my head I can think of 2 options:

* Rename the property to indicate better which sampling it is (but then it may be quite lengthy)
* Mention in the docs that it is only for the resources tracked (but user can miss the docs)

We could also remove it from `DdSdkReactNativeConfiguration` and rely on the call `DdRumResourceTracking#startTracking` done by user with the necessary sampling value, but this is not really an option, because:

1) `DdRumResourceTracking` is not exported to the public API currently.
2) Expecting user to call `DdRumResourceTracking#startTracking` makes `DdSdkReactNativeConfiguration#trackResources` useless.

Ideally, when we get rid of the bridges we could create a RUM-specific, Trace-specific, Log-specific, etc. configuration objects, but this for the future.

For now let's maybe rename it to be more specific. WDYT?